src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml.cs

```diff
@@ -140,5 +140,35 @@ private void OnButtonPreviewMouseMove(object sender, MouseEventArgs e)
         }
 
         #endregion
+
+        private void OnClassNamePreviewMouseDown(object sender, MouseButtonEventArgs e)
+        {
+            var senderButton = e.OriginalSource as FrameworkElement;
+            if (senderButton == null)
+            {
+                return;
+            }
+
+            var searchElementVM = senderButton.DataContext as NodeSearchElementViewModel;
+            if (searchElementVM == null)
+            {
+                return;
+            }
+
+            int lastIndex =
+                searchElementVM.FullName.LastIndexOf(Configuration.Configurations.CategoryDelimiterString,
+                    StringComparison.Ordinal);
+            string selectedClassWithCategory;
```

Comment:
move the initialization to the beginning of this method. say selectedClassWithCategory = string.empty